Multihost/basic-attestation/test.sh

```diff
@@ -310,9 +310,7 @@ _EOF"
         rlRun "$TESTDIR/keylime-bad-script.sh"
         rlRun "limeWaitForAgentStatus $AGENT_ID '(Failed|Invalid Quote)'"
         # give the revocation notifier a bit more time to contact the agent
-        rlRun "rlWaitForCmd 'tail $(limeAgentLogfile) | grep -q \"Executing revocation action\"' -m 20 -d 1 -t 20"
-        rlRun "tail $(limeAgentLogfile) | grep 'Executing revocation action local_action_modify_payload'"
-        rlRun "tail $(limeAgentLogfile) | grep 'A node in the network has been compromised: ${AGENT_IP}'"
+        rlRun "rlWaitForCmd 'tail $(limeAgentLogfile) | grep -q \"A node in the network has been compromised: ${AGENT_IP}\"' -m 20 -d 1 -t 20"
```

Comment:
Are those necessary? I removed them because the line tested in `rlWaitForCmd` is printed by the script, so there is no need to check for the other lines to make sure the script was executed.

I'll replace `$(limeAgentLogfile)` with `\$(limeAgentLogfile)` on tail command.

Reply:
Maybe not necessary, but I assume that some log messages are important and we would like to make sure they are logged (like the message about an agent failing attestation). OTOH, we do not need to have these in all tests, one test may be enough.